xdsl/irdl/attributes.py

```diff
@@ -91,9 +92,25 @@ def constr(constr: AttrConstraint) -> AttrConstraint:
 # |_|   \__,_|_|  \__,_|_| |_| |_/_/   \_\__|\__|_|
 #
 
-_A = TypeVar("_A", bound=Attribute)
 
-ParameterDef = Annotated[_A, IRDLAnnotations.ParamDefAnnot]
+# Field definition classes for `@irdl_param_attr_definition`
```

Comment:
Make this a documentation instead of a comment?

Reply:
Done